client/src/App.js

```diff
@@ -12,7 +13,8 @@ function App() {
           <div id="nav">
             <ResponsiveAppBar></ResponsiveAppBar>
           </div>
-          <Routes >
+          <Routes>
+            <Route path='/' element={<SignIn/>}></Route>
```

Comment:
Let's create a different route for the <SignIn> and create a home page with the '/' root path